app/steps/ui/deceased/dod/template.html

```diff
@@ -35,17 +35,20 @@
             {
                 classes: "govuk-input--width-2" + (" govuk-input--error" if fields["dod-day"].error or fields["dod-date"].error),
                 name: "day",
-                value: fields["dod-day"].value
+                value: fields["dod-day"].value,
+                autocomplete: "dod-day"
```

Comment:
Autocomplete attribute is 'bday-day', similarly with month and year

Reply:
Removed this as it doesn't make sense to have an autocomplete on the DoB of someone who died :)